### PR TITLE
Don't add attribute "class" if passed array of nulled classes in `Html::addCssClass()`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 ## 3.2.1 under development
 
-- New #173: Add class for tag 'html' and method `Html::html()` (@dood-)
-- Chg #179: Replace constant `PHP_EOL` to `"\n"` (@vjik) 
+- New #173: Add class for tag `html` and method `Html::html()` (@dood-)
+- Chg #179: Replace constant `PHP_EOL` to `"\n"` (@vjik)
+- Enh #180: Don't add attribute "class" if passed array of nulled classes in `Html::addCssClass()` (@vjik)
```
Comment (arogachev) [inline CHANGELOG.md:7]:
```suggestion
- Enh #180: Don't add "class" attribute in `Html::addCssClass()` if passed array contains null classes only (@vjik)
```

File: src/Html.php [hunk 2]
```diff
@@ -1666,6 +1666,12 @@ public static function addCssClass(array &$options, string|array|null $class): v
         if ($class === null) {
             return;
         }
+        if (is_array($class)) {
+            $class = array_filter($class, static fn ($c) => $c !== null);
```
Comment (arogachev) [inline src/Html.php:1670]:
```suggestion
            $class = array_filter($class, static fn (mixed $c): bool => $c !== null);
```

File: tests/common/HtmlTest.php [hunk 3]
```diff
@@ -883,8 +883,12 @@ public function dataAddCssClass(): array
...
             10 => [['class' => 'test test2'], ['class' => 'test test'], 'test2'],
             11 => [['class' => 'test test2'], ['class' => 'test test2'], null],
-            12 => [['class' => ['test', 'test2']], ['class' => 'test test2'], [null]],
+            12 => [['class' => 'test test2'], ['class' => 'test test2'], [null]],
             13 => [['class' => ['t1', 't2', 't3', 't4']], ['class' => 't1 t2'], ['t3', null, 't4']],
+            14 => [['id' => 'w2'], ['id' => 'w2'], null],
+            15 => [['id' => 'w2'], ['id' => 'w2'], [null]],
+            16 => [['id' => 'w2', 'class' => 't1'], ['id' => 'w2'], 't1'],
+            17 => [['id' => 'w2', 'class' => ['t3', 2 => 't4']], ['id' => 'w2'], ['t3', null, 't4']],
         ];
```
Comment (arogachev) [inline tests/common/HtmlTest.php:892]: Maybe add test with multiple `null` values?

**Review by arogachev — APPROVED** [review]

File: tests/common/HtmlTest.php (hunk 3 above)
Comment (vjik) [reply tests/common/HtmlTest.php:893]: done
